rust/kcl-lib/src/std/chamfer.rs

```diff
@@ -22,18 +26,41 @@ pub async fn chamfer(exec_state: &mut ExecState, args: Args) -> Result<KclValue,
     let solid = args.get_unlabeled_kw_arg("solid", &RuntimeType::solid(), exec_state)?;
     let length: TyF64 = args.get_kw_arg("length", &RuntimeType::length(), exec_state)?;
     let tags = args.kw_arg_edge_array_and_source("tags")?;
+    let second_length = args.get_kw_arg_opt("secondLength", &RuntimeType::length(), exec_state)?;
+    let angle = args.get_kw_arg_opt("angle", &RuntimeType::angle(), exec_state)?;
+    let swap = args.get_kw_arg_opt("swap", &RuntimeType::bool(), exec_state)?;
+    // TODO: custom profiles not ready yet
+    // let custom_profile = args.get_kw_arg_opt("customProfile", &RuntimeType::sketch(), exec_state)?;
```

Comment:
Could you just add the code when it's ready?

```suggestion
    // TODO: custom profiles not ready yet
```